utils/build-script-impl

```diff
@@ -71,7 +71,7 @@ KNOWN_SETTINGS=(
     llvm-enable-assertions      "1"              "enable assertions in LLVM and Clang"
     swift-build-type            "Debug"          "the CMake build variant for Swift"
     swift-enable-assertions     "1"              "enable assertions in Swift"
-    swift-analyze-code-coverage "0"              "enable code coverage analysis in Swift"
+    swift-analyze-code-coverage "default"        "enable code coverage analysis in Swift (set to \`merged\` to merge profdata files and remove intermediaries)"
```

Comment:
What are the possible option values?  What does `default` mean?  Could we just use an explicit value, like `none`?